src/Jackett.Common/Definitions/zamundanet.yml

```diff
@@ -20,6 +20,11 @@
       20: Movies/DVD
       9: XXX
       49: XXX/Other
+      6: Audio
+      30: Audio/Other
+      29: Music/DVD-R
```

Comment:
`Music/DVD-R` & `Music/Hi-Res/Vinyl` are not valid torznab categories
see https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt#L608-L695

Reply:
Yeah, sorry about that. It is a copy-paste mistake I did 👍